src/components/comments-list/CommentsList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Comment from "../comment/Comment";
+
+export default function CommentsList({comments}) {
+    let commentsList = comments.map((item) => {
+        return (
+            <Comment key={item.id}
```

Comment:
```suggestion
            <Comment 
              key={item.id}
```
It's easier to read if you pass each prop on the new line